src/renderer/entities/governance/lib/governanceService.ts

```diff
@@ -185,93 +213,95 @@ async function getVotingFor(
   api: ApiPromise,
   tracksIds: TrackId[],
   addresses: Address[],
-): Promise<Record<Address, Record<TrackId, Voting>>> {
+  callback: (res?: Record<Address, Record<TrackId, Voting>>) => void,
```

Comment:
IMO it's incorrect to mix pull and push access models, fetch and subscription should be separated. Common parts like mappings can be moved to helpers function.